webcompat/webhooks/helpers.py

```diff
@@ -38,13 +40,33 @@ def extract_browser_label(metadata_dict):
     # Only proceed if browser looks like "FooBrowser 99.0"
     if browser and re.search(r'([^\d]+?)\s[\d\.]+', browser):
         browser = browser.lower()
-        browser = browser.rsplit(' ', 1)[0]
         browser = browser.encode('utf-8')
+        # Remove parenthesis from the name
         browser = browser.translate(None, '()')
-        dash_browser = '-'.join(browser.split())
-        return 'browser-{name}'.format(name=dash_browser)
+        # Before returning a label, we need to clean up a bit
+        label_data = browser.split(' ')
+        # Let's focus on the known browser.
+        name = label_data[0]
+        if name not in BROWSERS:
+            browser_name = 'fixme'
+        else:
+            browser_name = label_data[0]
+        # Let's find a type for the browser.
+        remainder = label_data[1:]
+        browser_type = None
+        if 'mobile' in remainder:
+            browser_type = 'mobile'
+        if 'tablet' in remainder:
+            browser_type = 'tablet'
+        if browser_type:
+            dash_browser = '{browser_name}-{browser_type}'.format(
```

Comment:
In theory it would be possible to have an input like `LOL Mobile 55.0`, which would result in `browser-fixme-mobile` --- do we want this?

Reply:
A missing test! ❤️ 
Thanks!